schedule/yast/encryption/cryptlvm_iscsi.yaml

```diff
@@ -0,0 +1,49 @@
+---
+name: cryptlvm_iscsi
+description: >
+  Conducts installation on iSCSI device relying on iBFT with encrypted LVM.
+vars:
+  DESKTOP: gnome
+  ENCRYPT: 1
+  IBFT: 1
+  LVM: 1
+  MAX_JOB_TIME: 14400
+  NBF: iqn.2016-02.openqa.de:for.openqa
+  NICTYPE: user
+  TIMEOUT_SCALE: 3
```

Comment:
I can see that `TMIEOUT_SCALE` and `MAX_JOB_TIME` were added as a workarounds. So I would not set them here, but rather in job group, so we don't need to touch schedule to get rid of those. I'm not even sure if we still need those, will file a ticket to check it.